src/main/java/Duke.java

```diff
@@ -1,10 +1,131 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String LONG_LINE = "____________________________________________________________";
+    public static final int MAX_ARRAY_SIZE = 100;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printGreetings();
+        Task[] taskList = new Task[100];
+        String line = "Start";
+        Scanner in = new Scanner(System.in);
+        int listCount = 0;
+        while (!line.equals("bye")) {
+            line = in.nextLine();
+            if (!line.equals("bye")) {
+                listCount = handleUserInputs(line, taskList, listCount);
+            }
+            System.out.println(LONG_LINE);
+        }
+        printExit();
+        return;
+    }
+
+
+    private static void printList(Task[] s, int listCount) {
```

Comment:
I think the naming of the Task array 's' can be more descriptive by using a noun (for instance, 'taskList' as you put in line 34) for the variable name so that it is more understandable and readable